frontend/src/metabase/admin/databases/containers/DatabaseConnectionModal.tsx

```diff
@@ -90,11 +92,18 @@ export const DatabaseConnectionModalInner = ({
       }}
     >
       <DatabaseEditConnectionForm
-        database={database}
+        database={database as any}
```

Comment:
Do we need this `as any` here? I think we modify `mapStateToProps` to remove using the `Database` from metabase lib, and just use `metabase-types/api`. Unless I'm missing something downstream.

Reply:
Fixed!